### ISSUE TITLE
Auth error is thrown

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#645]
```
RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: undefined
    at ServerResponse.writeHead (_http_server.js:208:11)
    at ServerResponse._implicitHeader (_http_server.js:199:8)
    at write_ (_http_outgoing.js:593:9)
    at ServerResponse.end (_http_outgoing.js:714:5)
    at onFail (/src/node_modules/resolve-auth/dist/createAuthOptions.js:31:13)
    at Strategy.failWithLog (/src/node_modules/passport-azure-ad/lib/oidcstrategy.js:1484:15)
    at async.waterfall (/src/node_modules/passport-azure-ad/lib/oidcstrategy.js:650:21)
    at /src/node_modules/async/lib/async.js:52:16
    at /src/node_modules/async/lib/async.js:696:30
    at /src/node_modules/async/lib/async.js:167:37
    at Immediate.<anonymous> (/src/node_modules/async/lib/async.js:1206:34)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)
    at processImmediate (timers.js:649:5)
```

It is possible that the problem in on the following line: https://github.com/reimagined/resolve/blob/master/packages/resolve-auth/src/createAuthOptions.js#L28. Add condition to check whether the `status` variable is defined.

**Comment by TatyanaRyzh** [comment]
Hi timbset,
 
We've reproduced your scenario of using `resolve-auth` package and will fix it as soon as possible.
 
Thank you for helping us improve reSolve!

**Comment by IhostVlad** [comment]
Implemented in [V0.12.3](https://github.com/reimagined/resolve/releases/tag/V0.12.3)
